src/share/classes/com/sun/btrace/runtime/instr/SynchronizedInstrumentor.java

```diff
@@ -74,6 +74,18 @@ public void visitInsn(int opcode) {
         } else if (opcode == MONITOREXIT) {
             onBeforeSyncExit();
         }
+        switch (opcode) {
```

Comment:
Yes, but do like that,  it would change the ErrorReturnInstrumentor  logic of ERROR KIND Type, it is OK?